src/Tribe/Tickets_View.php

```diff
@@ -811,19 +811,19 @@ public function has_ticket_attendees( $event_id, $user_id = null ) {
 	 *
 	 * @since 4.2
 	 * @since 4.10.8 Deprecated the 3rd parameter (whether or not to use 'plurals') in favor of figuring it out per type.
+	 * @since TBD Updated signature to accept RSVP and ticket count arguments and removed deprecated 3rd parameter.
 	 *
 	 * @param int      $event_id   The Event ID we're checking.
 	 * @param int|null $user_id    An optional User ID.
 	 * @param null     $deprecated Deprecated argument.
 	 *
 	 * @return string
 	 */
-	public function get_description_rsvp_ticket( $event_id, $user_id = null, $deprecated = null ) {
+	public function get_description_rsvp_ticket( $event_id, $user_id = null, $rsvp_count = null, $ticket_count = null ) {
```

Comment:
Here you add one argument per RSVP count, one for Tickets' count: how will this scale with more type of tickets?

It looks like you're not using this function in the modified code: why change it?

Reply:
Yes, you are correct. I have skipped using this function within the template as I found the function name is specific to `rsvp` but used in the ticket context as well. 

The changes were made before I updated the template. Now, they don't have any impact on the changes that are made from this PR. I was thinking of reverting it as well. But I kept it in place as separating the dependency is still a kind of improvement from the existing one.

We won't need to scale it, as we are now building the labels within the template, as that feels more logical. 

This method is now only used here: https://github.com/the-events-calendar/event-tickets/blob/d2b8d1aac35c9ed6c30531758ee09317e650ba4e/src/views/tickets/orders.php#L169

Ideally, we should move everything out of the template and then set as template vars, but the same template is used for legacy views as well, so I didn't want to increase the scope of the original changes within this PR.

Reply:
I have reverted the changes, made to this one. I think it makes more sense to skip updating it as we are not using it within this PR.